integration/docker/cpu_test.go

```diff
@@ -16,9 +16,9 @@ import (
 )
 
 const (
-	sharesSysPath     = "/sys/fs/cgroup/cpu,cpuacct/cpu.shares"
-	quotaSysPath      = "/sys/fs/cgroup/cpu,cpuacct/cpu.cfs_quota_us"
-	periodSysPath     = "/sys/fs/cgroup/cpu,cpuacct/cpu.cfs_period_us"
+	sharesSysPath     = "/sys/fs/cgroup/cpu/cpu.shares"
```

Comment:
Do these paths need to be under /proc when agent is run as init, based on your agent PR?

Reply:
nop, libcontainer mounts cgroups at /sys/fs